flickfinder-backend/src/main/java/com/flickfinder/flickfinderbackend/controllers/MovieController.java

```diff
@@ -42,16 +42,20 @@ public Flux<Movie> getTrendingMovies() {
     }
   
 //    @GetMapping("/movies")
-//    public List<Movie> getMovies() {
-//        return;
-//        // Return a list of available movies
-//    }
-//
-//
+    public List<Movie> getMovies() {
+        return movieService.getMovies();
```

Comment:
getMovies() doesn't exist in the movieService in java so this will throw a compiling error.